lib/draper/query_methods/load_strategy.rb

```diff
@@ -0,0 +1,25 @@
+module Draper
+  module QueryMethods
+    module LoadStrategy
+      def self.new(name)
+        const_get(name.to_s.camelize).new
+      end
+
+      def allowed?(method)
+        raise NotImplementedError
+      end
+
+      class ActiveRecord
+        def allowed?(method)
+          ::ActiveRecord::Relation::VALUE_METHODS.include? method
+        end
+      end
+
+      class Mongo
```

Comment:
Should this technically be `Mongoid`, as it's the actual ORM we are referring to? If we ever wanted to add support for say [MongoMapper](http://mongomapper.com/), we'd need to be able to specify that.

Reply:
Yes, it's the referred ORM. Good catch.